packages/boxel-ui/addon/src/components/resizable-panel-group/handle.gts

```diff
@@ -90,11 +90,11 @@ export default class Handle extends Component<Signature> {
       }
 
       .resize-handle.horizontal {
-        cursor: col-resize;
+        cursor: unset;
```

Comment:
please let's not change the cursor

Reply:
Thank you @habdelra for the feedback, will revert the changes as the intention was to get rid of the visual pointer